tests/test_wasting.py

```diff
@@ -0,0 +1,1062 @@
+        assert pd.isnull(person['un_am_recovery_date'])
+        if am_state_expected == 'MAM':
+            assert df.at[person_id, 'un_clinical_acute_malnutrition'] == 'MAM'
+        else:
+            assert df.at[person_id, 'un_clinical_acute_malnutrition'] == 'SAM'
+
+        # Check that there is a natural recovery event scheduled:
+        #  Wasting_FullRecovery_Event if this person has MAM, Wasting_RecoveryToMAM_Event if this person has SAM
+        if am_state_expected == 'MAM':
+            recov_event_type = Wasting_FullRecovery_Event
+        else:  # am_state_expected == 'SAM'
+            recov_event_type = Wasting_RecoveryToMAM_Event
+        recov_event_tuple = [event_tuple for event_tuple in sim.find_events_for_person(person_id)
+                             if isinstance(event_tuple[1], recov_event_type)][0]
+        date_of_scheduled_recov = recov_event_tuple[0]
+        recov_event = recov_event_tuple[1]
+        assert date_of_scheduled_recov > sim.date
+
+        # Check no progression to sev wasting is scheduled
+        progress_event_tuple = next((event_tuple for event_tuple in sim.find_events_for_person(person_id)
+                                     if isinstance(event_tuple[1], Wasting_ProgressionToSevere_Event)), None)
+        assert not progress_event_tuple
+
+        # Run the natural recovery event:
+        sim.date = date_of_scheduled_recov
+        recov_event.apply(person_id)
+
+        # Check properties of this individual, if recovered from MAM should be well, if recovered from SAM should be MAM
+        person = df.loc[person_id]
+        if am_state_expected == 'MAM':  # with moderate wasting
+            assert person['un_WHZ_category'] == 'WHZ>=-2'
+            assert person['un_am_MUAC_category'] == '>=125mm'
+            assert not person['un_am_nutritional_oedema']
+            assert person['un_clinical_acute_malnutrition'] == 'well'
+            assert not person['un_sam_with_complications']
+            assert person['un_am_recovery_date'] == sim.date
+            assert pd.isnull(person['un_sam_death_date'])
+        else:  # am_state_expected == 'SAM' with moderate wasting
+            assert not person['un_am_nutritional_oedema']
+            assert df.at[person_id, 'un_clinical_acute_malnutrition'] == 'MAM'
+            assert not person['un_sam_with_complications']
+            assert pd.isnull(person['un_am_recovery_date'])
+            assert pd.isnull(person['un_sam_death_date'])
+
+
+def test_tx_recovery_to_MAM_severe_acute_malnutrition_without_complications(tmpdir):
+    """ Check the onset of symptoms with uncomplicated SAM, check recovery to MAM with tx when
+    the progression to severe wasting is certain, hence no natural recovery from moderate wasting,
+    the natural death due to SAM is certain, hence no natural recovery from severe wasting,
+    and check natural death canceled w\ tx and symptoms resolved when recovered to MAM with tx. """
```

Comment:
I really appreciate the docstrings. Very small thing - but could you write out in full to avoid the short hands that some people won't follow (e.g. w/, tx, etc/.)